contracts/CometCore.sol

```diff
@@ -56,6 +56,13 @@ abstract contract CometCore is CometConfiguration, CometStorage, CometMath {
     /// @dev The scale for factors
     uint64 internal constant FACTOR_SCALE = 1e18;
 
+    /// @dev The storage slot for reentrancy guard flags
+    bytes32 internal constant COMET_REENTRANCY_GUARD_FLAG_SLOT = bytes32(keccak256("comet.reentrancy.guard"));
+
+    /// @dev The reentrancy guard status
+    uint256 internal constant COMET_REENTRANCY_GUARD_NOT_ENTERED = 0;
+    uint256 internal constant COMET_REENTRANCY_GUARD_ENTERED = 2;
```

Comment:
I wonder if defining these as constants rather than inlining has an effect on bytecode size. I think not, but would be good to verify.

Reply:
hmmm interestingly that I tried both way the compiled bytescode size are identical 🤔